### ISSUE TITLE
accuracy of the image unimodal network on AVMNIST

### ISSUE THREAD

**Issue by zhengqigao** [juanmanpr/mfas#13]
Hi, when I run the code training the unimodal image network (LeNet5 structure, as depicted in the paper) on the disturbed MNIST (25% energy removed), I obtain an accuracy ~53% instead of ~74% as described in the paper. I also tested in an extreme case when only 1% energy is removed, which gives an accuracy 95% as expected. This implies the problem lies in my dataset, instead of training settings, I believe. 

I was wondering what might be the issue? Or have you ever came across this problem before? Thanks for your time.
